CHANGELOG.md

```diff
@@ -15,7 +15,7 @@
 
 #### Improvements
 
-- Disables sql simplification when sort is performed after limit. 
+- Disables sql simplification when sort is performed after limit.
```

Comment:
@sfc-gh-aalam do we have a ticket to track the documentation for multi-threading session object, i think you will want to mention this behavior in the doc on release

Reply:
created: https://snowflakecomputing.atlassian.net/browse/SNOW-1758759